### ISSUE TITLE
Support of jhipster >=6

### ISSUE THREAD

**Issue by shwabnitz** [hipster-labs/generator-jhipster-entity-audit#116]
Is a compatibility of jhipster 6 planned? The features of this module are important for business applications.

**Comment by deepu105** [comment]
We don't have the bandwidth at the moment, if you are willing to contribute
and do a PR we will be happy to review

Thanks & Regards,
Deepu

**Comment by murdos** [comment]
FWIW master nearly supports version 6, the main issue currently is fixing the integration tests.

**Comment by shwabnitz** [comment]
Thank you @murdos . Installing master (which is default) I tried running yeoman:  `yo jhipster-entity-audit` and got the error below. In may this year @loverto  commited d222d15 which seemed to revert support of Jhipster 6.


```
yo jhipster-entity-audit
events.js:174
      throw er; // Unhandled 'error' event
      ^

TypeError: this.getJhipsterAppConfig is not a function
    at module.exports.readConfig (.../.npm/lib/node_modules/generator-jhipster-entity-audit/generators/app/index.js:23:33)
    at Object.<anonymous> (.../.npm/lib/node_modules/generator-jhipster-entity-audit/node_modules/generator-jhipster/node_modules/yeoman-generator/lib/index.js:424:27)
    at .../.npm/lib/node_modules/generator-jhipster-entity-audit/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at .../.npm/lib/node_modules/generator-jhipster-entity-audit/node_modules/run-async/index.js:24:19
    at self.env.runLoop.add.completed (.../.npm/lib/node_modules/generator-jhipster-entity-audit/node_modules/generator-jhipster/node_modules/yeoman-generator/lib/index.js:425:13)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
Emitted 'error' event at:
    at Immediate.setImmediate (.../.npm/lib/node_modules/generator-jhipster-entity-audit/node_modules/generator-jhipster/node_modules/yeoman-generator/lib/index.js:433:22)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

**Comment by murdos** [comment]
Good catch... It seems that things get messed up when I merged both PR https://github.com/hipster-labs/generator-jhipster-entity-audit/pull/109 and https://github.com/hipster-labs/generator-jhipster-entity-audit/pull/111

@shwabnitz would you like to contribute a PR to fix the minimal jhispter version in package.json?

**Comment by yakari** [comment]
Hello, I submitted a PR (#119)  to enable support for JHipster 6 (tested with v6.4.1). It supports lazy loading. Hope this helps!

**Comment by poolebu** [comment]
Thank you @yakari for working on this. I'm in need of this module too, so let me know if you need any more help. And how could I know when is ready for use? I guess I'll be a first user

**Comment by murdos** [comment]
Issue fixed with #119 ; a new version should be released soon
